client/my-sites/customer-home/cards/notices/reader-first-posts/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { Card, Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+//import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+//import { NOTICE_READER_FIRST_POSTS } from 'calypso/my-sites/customer-home/cards/constants';
+
+const ReaderFirstPosts = () => {
+	const translate = useTranslate();
+
+	const clickButton = () => {
+		// @TODO: Add Tracks events.
+		//recordTracksEvent( 'calypso_home_reader_first_posts_nudge_click' );
+
+		page.redirect( '/discover?selectedTab=first-posts' );
```

Comment:
The redirect needs to go `firstposts` instead of `first-posts`.

```suggestion
		page.redirect( '/discover?selectedTab=firstposts' );
```

Reply:
Good catch, I updated that in the Tracks PR but not this one. 😆